recipe/meta.yaml

```diff
@@ -12,12 +12,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [win or not py2k]
+  number: 2
+  skip: True  # [win and py3k or win64 or not py2k]
```

Comment:
When you change things that will affect the build matrix, like the `skip` section, you need to rerender to get the change into the CIs configuration. See http://conda-forge.github.io/docs/conda_smithy.html